packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -709,6 +709,43 @@ describe("SharedString", () => {
 			// Verify that the changes were correctly received by the second SharedString
 			assert.equal(sharedString2.getText(), "hello friend");
 		});
+		
+		it("insert in middle of multibyte character", async () => {
```

Comment:
this snuck in here on accident, but i don't think it's harmful to include, or at least harmful enough to warrant splitting out to a separate PR.